Source/Plugins/EditorBase/PropertyEditors/GameObjectOverviewPropertyEditor.cs

```diff
@@ -120,6 +122,26 @@ protected void UpdateComponentEditors(GameObject[] values)
 			this.EndUpdate();
 		}
 
+		protected void UpdateAddComponentEditor(GameObject[] values)
+		{
+			this.BeginUpdate();
+
+			if (this.addComponentEditor == null)
+			{
+				this.addComponentEditor = new AddComponentPropertyEditor();
+				this.addComponentEditor.Getter = this.GetValue;
+				this.addComponentEditor.Setter = this.SetValue;
+				this.AddPropertyEditor(this.addComponentEditor, this.ChildEditors.Count);
+			}
+			else
+			{
+				this.RemovePropertyEditor(this.addComponentEditor);
+				this.AddPropertyEditor(this.addComponentEditor, this.ChildEditors.Count);
```

Comment:
Why are you doing the add-remove here? Is it to ensure the editor remains at the bottom as new editors are added? If that's the case, please add a comment above those two lines, so it's clear to the reader.